packages/dscc-gen/src/config.ts

```diff
@@ -99,16 +99,16 @@ const commonQuestions: Array<Question<CommonConfig>> = [
 
 const vizQuestions: Array<Question<VizConfig>> = commonQuestions.concat([
   {
-    name: 'devBucket',
+    name: 'devDirectory',
     type: 'input',
-    message: 'What is your dev bucket?',
+    message: 'What is your dev directory?',
```

Comment:
How about "dev directory on GCS"? (here and below)

Reply:
I changed it to "GCS dev directory"